packages/circuit-ui/components/Spinner/Spinner.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Status, Props, Story } from '../../../../.storybook/components';
+import * as Stories from './Spinner.stories';
+
+<Meta of={Stories} />
+
+# Spinner
+
+<Status.InReview />
+
+The Spinner component is used to visually represent the loading state of a UI component.
+
+<Story of={Stories.Base} />
+<Props />
+
+## Usage guidelines
+
+> This section is a stub. Contributions welcome!
```

Comment:
What do you think about this Wikipedia-inspired approach, generally?

I like it because at least we have scaffolding the right docs and sections (for linking from component A to B for example, in this case), and then we can expand on them later (or maybe contributors will suddenly turn up 🥺)

Reply:
I like the idea for missing docs for existing components. I wouldn't release new components without docs 😄

Reply:
Yup, 100% agree here 🙂